Source/ChartIQScriptManager.swift

```diff
@@ -456,12 +467,14 @@ internal class ChartIQScriptManager: ChartIQScriptManagerProtocol {
   /// - Returns: The String Object that contains a JS script for evaluate in the WebView.
   internal func getScriptForSetStudyParameters(_ study: ChartIQStudy, parameters: [String: String]) -> String {
     var script = getStudyDescriptorScript(study.fullName) +
-      "var helper = new CIQ.Studies.DialogHelper({sd:selectedSd,stx:stxx}); " + "var isFound = false; " +
+      "var helper = new CIQ.Studies.DialogHelper({sd:selectedSd,stx:" + chartIQJsObject + "}); " + "var isFound = false; " +
       "var newInputParameters = {}; " + "var newOutputParameters = {}; " + "var newParameters = {}; "
     parameters.forEach { parameter in
       script += getUpdateStudyParametersScript(parameter.key, value: parameter.value)
     }
-    script += "helper.updateStudy({inputs:newInputParameters, outputs:newOutputParameters, parameters:newParameters}); console.log(JSON.stringify(newParameters)) "
+    script += "helper.updateStudy({inputs:newInputParameters, outputs:newOutputParameters, parameters:newParameters}); console.log(JSON.stringify(newParameters));"
```

Comment:
Do you need the console logging?

Reply:
Probably not, users can add it back in if they want.